### DISCUSSION TITLE
Webpage card improvements

### DISCUSSION THREAD

**Discussion by and7ey** [home-assistant/frontend#19085] [Other feature requests]
Please consider the following ideas to make webpage card better:
- automatic webpage refresh (with given timeout)
- pass css/vars theme to iframe (at least - dark/light mode) 
- disable scrolling

**Comment by ildar170975** [comment]
> dark/light mode

How a webpage is supposed to process it?

**Reply by and7ey** [reply]
change css style accordingly
